Firestore/Example/App/macOS_example/AppDelegate.m

```diff
@@ -35,7 +35,6 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
 
   // do the timestamp fix
   FIRFirestoreSettings *settings = db.settings;
-  settings.timestampsInSnapshotsEnabled = true;
   db.settings = settings;
```

Comment:
This change seems unrelated?

Reply:
This shows up as a deprecation warning while building the macOS_example, which was the primary means by which I verified this change was correct.

I can move this into a separate cleanup grab-bag PR if you prefer.

Reply:
No, I think that would be overkill (applies to other changes of that kind in this PR). As long as this is done on purpose, I'm fine with it being the part of the same PR.